server/server.js

```diff
@@ -12,12 +12,18 @@ app.use(cors());
 
 // parse the request body
 app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
```

Comment:
Why extended true? I didn't think we needed that functionality

Reply:
Good catch. When I was messing around with Multer stuff I thought that I needed it but it seems we don't.